.pre-commit-config.yaml

```diff
@@ -23,7 +23,7 @@ repos:
     -   id: blacken-docs
         additional_dependencies: [black==22.10]
 -   repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
Anything thing wrong with 10.1 or is there some feature we need for 12.0?

Reply:
pre-commit fails to install isort 5.10.1 for [this reason](https://stackoverflow.com/questions/75269700/pre-commit-fails-to-install-isort-5-11-4-with-error-runtimeerror-the-poetry-co), but it can install isort 5.12.0 as suggested in the answer.

Reply:
I am pushing this isort change to main. Without the version upgrade, pre-commit stopped working.